### PR TITLE
add content to IAM part

### PR DESCRIPTION
**Pull request by marcom4rtinez**
@sevigrimm review, please also add comment for points marked in file

### CODE REVIEW

File: best-practices/content/10_IAM.tex [hunk 1]
```diff
@@ -1,13 +1,128 @@
 \chapter{\acrfull{iam}}
 
-\section{IAM Varianten}
+\section{\acrshort{iam} Varianten}
+Eine \acrfull{iam} ist ein zentrales System welches die Entitäten, wie Benutzeraccounts und Computer, erfassen kann. Zudem können Berechtigungen auf Entitäten in einer Domäne verwaltet.\\
 
+%TODO: style correct @sevigrimm should use quote or some fancy style?
+\textbf{Nun kommt die Frage auf: Geht es nicht ohne?}\\
+Es ist grundsätzlich möglich ohne ein IAM System zu arbeiten, wobei der Aufwand für das Management der Userverwaltung auf allen Maschinen schnell den Rahmen des Budgets sprengt.
+Der allgemeine Konsenus ist dass ab 15 aktiven Bentuzern ein IAM verwendet werden soll.\\
```
Comment (sevigrimm) [inline best-practices/content/10_IAM.tex:9]:
```suggestion
Der allgemeine Konsenus ist, dass ab 15 aktiven Bentuzern ein IAM verwendet werden soll.\\
```

File: best-practices/content/10_IAM.tex [hunk 2]
```diff
@@ -1,13 +1,128 @@
 \chapter{\acrfull{iam}}
 
-\section{IAM Varianten}
+\section{\acrshort{iam} Varianten}
+Eine \acrfull{iam} ist ein zentrales System welches die Entitäten, wie Benutzeraccounts und Computer, erfassen kann. Zudem können Berechtigungen auf Entitäten in einer Domäne verwaltet.\\
 
+%TODO: style correct @sevigrimm should use quote or some fancy style?
+\textbf{Nun kommt die Frage auf: Geht es nicht ohne?}\\
```
Comment (sevigrimm) [inline best-practices/content/10_IAM.tex:7]:
```suggestion
\textbf{Braucht es ein IAM?}\\
```

File: best-practices/content/10_IAM.tex [hunk 3]
```diff
@@ -1,13 +1,128 @@
 \chapter{\acrfull{iam}}
 
-\section{IAM Varianten}
+\section{\acrshort{iam} Varianten}
+Eine \acrfull{iam} ist ein zentrales System welches die Entitäten, wie Benutzeraccounts und Computer, erfassen kann. Zudem können Berechtigungen auf Entitäten in einer Domäne verwaltet.\\
```
Comment (sevigrimm) [inline best-practices/content/10_IAM.tex:4]:
```suggestion
Ein \acrfull{iam} ist ein zentrales System welches die Entitäten, wie Benutzeraccounts und Computer, erfassen kann. Zudem können Berechtigungen auf Entitäten in einer Domäne verwaltet.\\
```

File: best-practices/content/10_IAM.tex [hunk 4]
```diff
@@ -1,13 +1,128 @@
...
+    \end{multicols}
+\end{minipage}
+%TODO: @sevigrimm ergänzungen?
+%TODO: Add glossary for DHCP DNS NTP CA
+
+\vspace{1cm}
+Wie bei vielem ist es wichtig das richtige Produkt für seine Umgebung zu finden. Als nächstes werden zwei, im Markt beliebte Produkte vorgestellt.
+
+\subsection{Active Directory}
+Active Directory ist ein Produkt von Microsoft und kann exklusiv auf einem Microsoft Windows Server installiert werden. Active Directory ist einfach zu installieren, wobei die Konfiguration manchmal Herausforderungen birgt.\\
```
Comment (sevigrimm) [inline best-practices/content/10_IAM.tex:51]:
```suggestion
Active Directory ist ein Produkt von Microsoft und kann exklusiv auf einem Microsoft Windows Server installiert werden. Active Directory ist einfach zu installieren, wobei die Konfiguration Herausforderungen mit sich bringen kann.\\
```

File: best-practices/content/10_IAM.tex [hunk 5]
```diff
@@ -1,13 +1,128 @@
...
+        \begin{table}[H]
+            \begin{center}
+                \textbf{Vorteile:}
+                \begin{itemize}
+                    \item zentrales Accountmanagement
+                    \item zentrales Accessmanagement
+                    \item schnelle Skalierung Server/Client Umgebung
+                    \item single Source of Truth\footnote{\href{https://de.wikipedia.org/wiki/Single\_Point\_of\_Truth}{https://de.wikipedia.org/wiki/Single\_Point\_of\_Truth}}
+                    \item Policysystem zur Konfiguration von Client
+                    \item meist gebündelt mit nützlichen Features\footnote{DHCP, DNS, NTP, CA etc. (Produktabhängig)}
```
Comment (sevigrimm) [inline best-practices/content/10_IAM.tex:24]: \acrfull{} für DHCP, DNS, NTP, CA?

File: best-practices/content/10_IAM.tex [hunk 6]
```diff
@@ -1,13 +1,128 @@
...
+%TODO: @sevigrimm ergänzungen?
+%TODO: Add glossary for DHCP DNS NTP CA
+
+\vspace{1cm}
+Wie bei vielem ist es wichtig das richtige Produkt für seine Umgebung zu finden. Als nächstes werden zwei, im Markt beliebte Produkte vorgestellt.
+
+\subsection{Active Directory}
+Active Directory ist ein Produkt von Microsoft und kann exklusiv auf einem Microsoft Windows Server installiert werden. Active Directory ist einfach zu installieren, wobei die Konfiguration manchmal Herausforderungen birgt.\\
+
+Active Directory ist mit mehr als 80\% Marktanteil der Marktführer. Die Active Directory Funktion wird mit DNS, DHCP und NTP gebündelt. Das Produkt bietet alle grundlegenden Funktionen wie User-, Geräteverwaltung, DHCP, DNS, NTP und viele weitere einfache, bis zu komplexe Funktionen. Vorallem in Windowsnetzwerken ist die Integration sehr gut und intuitiv.\\
```
Comment (sevigrimm) [inline best-practices/content/10_IAM.tex:53]:
```suggestion
Active Directory ist mit mehr als 80\% Marktanteil der Marktführer. 
Das Produkt bietet alle grundlegenden Funktionen wie User-, Geräte- und Berechtigungsverwaltung.
Ausserdem lässt sich Active Directory integrieren mit anderen Windows Server Funktionen wie DHCP, DNS, NTP und weiteren externen Funktionen von anderen Anbietern. 
Vorallem in Windowsnetzwerken ist die Integration sehr gut und intuitiv.\\
```

File: best-practices/content/10_IAM.tex [hunk 7]
```diff
@@ -1,13 +1,128 @@
...
+
+Active Directory bietet in fast allen Fällen, was ein KMU braucht und viel mehr. Deshalb ist die Wahl von Active Directory für KMUs zu 99\% die richtige Wahle. Es gibt nur wenige sehr spezifische Fälle in denen ein anderes Produkt besseren Nutzen bringt.
```
Comment (sevigrimm) [inline best-practices/content/10_IAM.tex:55]:
```suggestion
Active Directory bietet in fast allen Fällen was ein KMU braucht und viel mehr. Deshalb ist die Wahl von Active Directory für KMUs zu meist die richtige Wahle. Es gibt nur wenige sehr spezifische Fälle in denen ein anderes Produkt besseren Nutzen erbringt. 
```

File: best-practices/content/10_IAM.tex [hunk 8]
```diff
@@ -1,13 +1,128 @@
...
+\subsection{Active Directory}
+Active Directory ist ein Produkt von Microsoft und kann exklusiv auf einem Microsoft Windows Server installiert werden. Active Directory ist einfach zu installieren, wobei die Konfiguration manchmal Herausforderungen birgt.\\
+
+Active Directory ist mit mehr als 80\% Marktanteil der Marktführer. Die Active Directory Funktion wird mit DNS, DHCP und NTP gebündelt. Das Produkt bietet alle grundlegenden Funktionen wie User-, Geräteverwaltung, DHCP, DNS, NTP und viele weitere einfache, bis zu komplexe Funktionen. Vorallem in Windowsnetzwerken ist die Integration sehr gut und intuitiv.\\
+
+Active Directory bietet in fast allen Fällen, was ein KMU braucht und viel mehr. Deshalb ist die Wahl von Active Directory für KMUs zu 99\% die richtige Wahle. Es gibt nur wenige sehr spezifische Fälle in denen ein anderes Produkt besseren Nutzen bringt. 
+
+
+\subsection{FreeIPA}
+FreeIPA ist eine Open-Source Implementation eines IAMs, entwickelt durch die Community und Upstream von Red Hat Identity Management. Exklusiv verfügbar auf RPM-based Linux Systemen.\\
```
Comment (sevigrimm) [inline best-practices/content/10_IAM.tex:59]:
```suggestion
FreeIPA ist eine Open-Source Implementation eines IAMs, entwickelt durch die Community und Upstream von Red Hat Identity Management. Exklusiv verfügbar auf RPM-basierten Linux Systemen.\\
```

Comment (sevigrimm) [reply best-practices/content/10_IAM.tex:59]: RPM erklären, evtl beispiele?

File: best-practices/content/10_IAM.tex [hunk 9]
```diff
@@ -1,13 +1,128 @@
...
+
+FreeIPA bietet von den Features einen ähnlichen Scope wie Active Directory, aber mit dem grossen Unterschied dass die Hauptzielgruppe Linux Server und Clients sind. Es wird auch User-, Geräteverwaltung, DHCP, DNS, NTP und viele weitere einfache, bis zu komplexe Funktionen.\\
```
Comment (sevigrimm) [inline best-practices/content/10_IAM.tex:61]:
```suggestion
FreeIPA bietet von den Features einen ähnlichen Umfang wie Active Directory, aber mit dem grossen Unterschied, dass die Hauptzielgruppe Linux Server und Clients sind. Es werden auch User-, Geräte- und Berechtigungsverwaltung, DHCP, DNS, NTP und viele weitere einfache, bis zu komplexe Funktionen angeboten.\\
```

File: best-practices/content/10_IAM.tex [hunk 10]
```diff
@@ -1,13 +1,128 @@
...
+Um die Angriffsfläche und somit das Risiko zu minimieren sollten die unten beschriebenen Konzepte beachtet und wenn möglich strikt implementiert werden.
 
-\section{Konzepte beim betreiben von IAMs}
 \subsection{Least Privilege}
+Das ``least privilege principle'' ist ein grundlegender Security Best Practice, denn man überall wieder findet.\\
+
+Das Prinzip besagt jeder Benutzer soll genau soviel Rechte haben, wie er für seine tägliche Arbeit benötigt. Alles weitere soll blockiert werden.
+Somit kann die Angriffsfläche bei Vorfällen massiv verringert werden.\\
+
+Für privilegierte Benutzer wie z.B. ein Systemadministrator sollen spezielle Accounts bereitgestellt werden.
```
Comment (sevigrimm) [inline best-practices/content/10_IAM.tex:82]:
```suggestion
Für privilegierte Benutzer wie z.B. ein Systemadministrator sollen zusätzliche, spezielle Accounts bereitgestellt werden.
```

File: best-practices/content/10_IAM.tex [hunk 11]
```diff
@@ -1,13 +1,128 @@
...
+Zusätzlich soll auch beachtet werden, dasses keine ``Masteraccounts'' gibt, welche alle Rechte besitzen.
+Den diese sind für Angreifer besonders interessant und für das Unternehmen gefährlich.
+
+
+\subsection{persönliche Accounts}
+Es muss versucht werden die Anzahl unpersönliche Accounts auf ein minimum beschränkt zu halten.
+Bei einem Incident sind die Logs ausschlaggebend.\\
+
+Arbeiten alle Benutzer mit dem Account ``Finanzen'' ist es schwer Rückschlüsse zu ziehen, wo kritische Aktionen oder ilegitime Zugriffe passiert sind.
+In diesem Fall wäre es klar der Verstoss ist mit dem Account ``Finanzen'' passiert, aber welcher der 10 Leute die den Account benutzen den Verstoss begangen hat ist schwer, bis unmöglich Rückzuschliessen.
```
Comment (sevigrimm) [inline best-practices/content/10_IAM.tex:94]:
```suggestion
In einem Vorfall wäre es klar, dass der Verstoss mit dem Account ``Finanzen'' passiert ist, aber welcher der 10 Personen die den Account benutzen den Verstoss begangen hat ist schwer, bis unmöglich Rückzuschliessen.
```

File: best-practices/content/10_IAM.tex [hunk 12]
```diff
@@ -1,13 +1,128 @@
...
+
+\subsection{Rollen als Berechtigungen}
+Unternehmungen sind ermutigt Jobprofile zu erstellen, um geeignetes Personal zu finden.
+Es ist ähnlich in der verteilung von Benutzerrechten.
```
Comment (sevigrimm) [inline best-practices/content/10_IAM.tex:98]:
```suggestion
Es ist ähnlich in der Verteilung von Benutzerrechten.
```

File: best-practices/content/10_IAM.tex [hunk 13]
```diff
@@ -1,13 +1,128 @@
...
+Eine Person ist angestellt um eine bestimmte Arbeit zu verrichten.
+Diese Zuteilung wird nun auf eine Rolle abgebildet.
+Die Rolle mit den Rechten erhalten alle Angestellten mit dem gleichen Auftrag.\\
+
+Das verteilen von Rollen erleichtert den Überblick der einzelnen Berechtigungen in der Unternehmung.
+Bei individual Berechtigungen verliert man schnell den Überblick.
+Mit administrativen Wechseln wie interne Umorientierungen aknn es dazu führen, dass Nutzer zu viele Berechtigungen für Ihren Job haben und somit das ``least privilege principle'' verletzen.
```
Comment (sevigrimm) [inline best-practices/content/10_IAM.tex:105]:
```suggestion
Mit administrativen Wechseln wie interne Umorientierungen kann es dazu führen, dass Nutzer zu viele Berechtigungen für Ihren Job haben und somit das ``least privilege principle'' verletzen.
```

File: best-practices/content/10_IAM.tex [hunk 14]
```diff
@@ -1,13 +1,128 @@
...
+Abschliessend, es sollte bei internen Wechseln lediglich die alte Rolle entfernt und die neue Rolle hinzugefügt werden.
```
Comment (sevigrimm) [inline best-practices/content/10_IAM.tex:106]:
```suggestion
Mit Rollen kann bei internen Wechseln lediglich die alte Rolle entfernt und die neue Rolle hinzugefügt werden.
```

File: best-practices/content/10_IAM.tex [hunk 15]
```diff
@@ -1,13 +1,128 @@
...
+
+Arbeiten alle Benutzer mit dem Account ``Finanzen'' ist es schwer Rückschlüsse zu ziehen, wo kritische Aktionen oder ilegitime Zugriffe passiert sind.
+In diesem Fall wäre es klar der Verstoss ist mit dem Account ``Finanzen'' passiert, aber welcher der 10 Leute die den Account benutzen den Verstoss begangen hat ist schwer, bis unmöglich Rückzuschliessen.
+
+\subsection{Rollen als Berechtigungen}
+Unternehmungen sind ermutigt Jobprofile zu erstellen, um geeignetes Personal zu finden.
+Es ist ähnlich in der verteilung von Benutzerrechten.
+Eine Person ist angestellt um eine bestimmte Arbeit zu verrichten.
+Diese Zuteilung wird nun auf eine Rolle abgebildet.
+Die Rolle mit den Rechten erhalten alle Angestellten mit dem gleichen Auftrag.\\
```
Comment (sevigrimm) [inline best-practices/content/10_IAM.tex:101]:
```suggestion
Die Rolle mit den Rechten erhalten alle Angestellten mit dem gleichen Auftrag.
Oftmals werden Rollen in einem IAM Gruppen genannt, welcher man Benutzer und andere Gruppen hinzufügen kann.\\
```

File: best-practices/content/10_IAM.tex [hunk 16]
```diff
@@ -1,13 +1,128 @@
...
+
+
+\subsection{Automatische Benutzerverwaltung}
+Es ist empfohlen die Benutzerverwaltung zu automatisieren oder einen klaren Prozess festzulegen.
+Dies betrifft vorallem die Benutzer, welche das Unternehmen verlassen.
+Die Accounts sollten bei Austritt deaktiviert und später gelöscht werden.
+
+
+\subsection{Zuständigkeitsbereich eines IAM}
+Es kann eine Herausforderung entstehen, wenn man alle seine Geräte an das IAM anbinden will.\\
```
Comment (sevigrimm) [inline best-practices/content/10_IAM.tex:116]: Evtl. no chli meh dezue sege?

File: best-practices/content/10_IAM.tex [hunk 17]
```diff
@@ -1,13 +1,128 @@
...
+Das verteilen von Rollen erleichtert den Überblick der einzelnen Berechtigungen in der Unternehmung.
+Bei individual Berechtigungen verliert man schnell den Überblick.
+Mit administrativen Wechseln wie interne Umorientierungen aknn es dazu führen, dass Nutzer zu viele Berechtigungen für Ihren Job haben und somit das ``least privilege principle'' verletzen.
+Abschliessend, es sollte bei internen Wechseln lediglich die alte Rolle entfernt und die neue Rolle hinzugefügt werden.
+
+
+\subsection{Automatische Benutzerverwaltung}
+Es ist empfohlen die Benutzerverwaltung zu automatisieren oder einen klaren Prozess festzulegen.
+Dies betrifft vorallem die Benutzer, welche das Unternehmen verlassen.
+Die Accounts sollten bei Austritt deaktiviert und später gelöscht werden.
```
Comment (sevigrimm) [inline best-practices/content/10_IAM.tex:112]:
```suggestion
Die Accounts sollten bei Austritt deaktiviert und und nach einer definierten Zeit gelöscht werden.
```

File: best-practices/content/10_IAM.tex [hunk 18]
```diff
@@ -1,13 +1,128 @@
...
-\subsection{Rollen bevor individul Berechtigungen}
+Im Idealfall sind alle Geräte an das IAM angebunden.
+Es gibt aber auch Instanzen, in welchen diese Lösung schlicht weg nicht wirtschaftlich vertretbar ist.
+Das geht in Ordnung.
+Bei solchen Geräten sollte sich überlegt werden, wie sie adäquat geschützt werden können.
 
-\subsection{Automatisches warten der Benutzer}
+\section{IAM Schlusswort}
+Ein IAM kann Agilität, Komfort und Sicherheit bringen.
+Mit den Vorteilen bringt ein IAM auch Risiken, wie Sicherheitsbedenken und Abhängigkeit für den produktiven Betrieb.\\
```
Comment (sevigrimm) [inline best-practices/content/10_IAM.tex:125]:
```suggestion
Mit den Vorteilen bringt ein IAM auch Risiken, wie Sicherheitsbedenken und Abhängigkeit für den produktiven Betrieb.
Daher sollte ein IAM bestmöglich geschützt werden.
Editorberechtigung im IAM sollten nur Personen zustehen, welche diese auch wirklich brauchen (least privilege).
```

File: best-practices/content/10_IAM.tex (hunk 6 above)
Comment (marcom4rtinez) [reply best-practices/content/10_IAM.tex:53]: @sevigrimm should I accept here or should we clarify a little?

File: best-practices/content/10_IAM.tex (hunk 8 above)
Comment (marcom4rtinez) [reply best-practices/content/10_IAM.tex:59]: @sevigrimm  kind of trivial? glossary maybe?

File: best-practices/content/10_IAM.tex [hunk 19]
```diff
@@ -1,13 +1,265 @@
...
+Falls die Passwordrichtline vorher bei weniger als 12 Zeichen war, müssen nun die Benuter noch aufgefordert werden, ihr Passwort zu ändern.
+Dies kann mit Powershell gemacht werden:
+\begin{lstlisting}
+    Get-ADUser -SearchBase "<OU der Benutzer>" -Filter * | Set-ADUser -ChangePasswordAtLogon $True
+
+    #Beispiel:
+    Get-ADUser -SearchBase "OU=Employees,OU=Users,OU=Lab,DC=ba,DC=lab" -Filter * | Set-ADUser -ChangePasswordAtLogon $True
+\end{lstlisting}
+
+Dies sollte mit dem Management abgeklärt werden und die Benutzer sollten frühzeitig benachrichtigt werden.
```
Comment (marcom4rtinez) [inline best-practices/content/10_IAM.tex:205]:
```suggestion
Dies sollte mit dem Management abgeklärt werden und die Benutzer sollten frühzeitig informiert werden.
```

File: best-practices/content/10_IAM.tex [hunk 20]
```diff
@@ -1,13 +1,265 @@
...
+Man kann auch zuerst die Mindestlänge in der Group Policy setzten und erst nach einem Zyklus, wenn alle Mitarbeitenden ihr Passwort auf die neue Länge gesetzt haben, die Option ``Maximum password age'' auf 0 setzten. \\
```
Comment (marcom4rtinez) [inline best-practices/content/10_IAM.tex:206]:
```suggestion
Man kann auch zuerst die Mindestlänge in der Group Policy setzen und erst nach einem Zyklus, wenn alle Mitarbeitenden ihr Passwort auf die neue Länge gesetzt haben, die Option ``Maximum password age'' auf 0 setzen. \\
```

File: best-practices/content/10_IAM.tex [hunk 21]
```diff
@@ -1,13 +1,265 @@
...
+    \includegraphics[width=0.7\linewidth]{../img/IAM/admin-password-policy.png}
+    \caption{Passwortrichtlinien Administratoren}
+\end{figure}
+Die Account Lockout Policy bleibt gleich.
+
+\subsection{Active Directory Auditing}
+Active Directory bietet die Möglichkeit, Änderungen and Gruppen und Benutzern im Event Log festzuhalten.
+Dies ist nützlich um nachvollziehen zu können, wann welche Änderungen gemacht wurden und zusammen mit Wazuh erkennen zu können, ob Änderungen gemacht wurden die nicht von Administratoren kamen.\\
+
+Dies Logs müssen mit einer Group Policy aktiviert werden.
```
Comment (marcom4rtinez) [inline best-practices/content/10_IAM.tex:221]:
```suggestion
Diese Logs müssen mit einer Group Policy aktiviert werden.
```

File: best-practices/content/10_IAM.tex [hunk 22]
```diff
@@ -1,13 +1,265 @@
...
+Im Group Policy Management existiert unter \textbf{Forest: <Domänen-Namen> $\rightarrow$ Domains $\rightarrow$ <Domänen-Namen> $\rightarrow$ Domain Controllers} die ``Default Domain Controllers Policy''.
+In dieser setzt man unter \textbf{Computer Configuration $\rightarrow$ Policies $\rightarrow$ Windows Settings $\rightarrow$ Security Settings $\rightarrow$ Local Policies $\rightarrow$ Audit Policy} in der Richtlinie ``Audit account Management'' folgende Einstellungen:\\
+\begin{figure}[H]
+    \centering
+    \includegraphics[width=0.7\linewidth]{../img/IAM/ad-account-management.png}
+    \caption{AD Audit Policy}
```
Comment (marcom4rtinez) [inline best-practices/content/10_IAM.tex:227]:
```suggestion
    \caption{Active Directory Audit Policy}
```

**Pull request merged**